components/resources/ai_chat_ui_strings.grdp

```diff
@@ -93,6 +93,9 @@
   <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_QWEN" desc="AI Chat intro message for the qwen model" formatter_data="webui=AiChat">
     Hi, I'm Leo. I'm a fully hosted AI assistant by Brave. I'm powered by Qwen 14B, a model created by Alibaba Cloud to be performant and applicable to many use cases. Qwen is Brave-hosted through our own secure infrastructure.<ph name="LINK_BEFORE">$1</ph>Learn more<ph name="LINK_AFTER">$1</ph>
   </message>
+  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_GEMMA" desc="AI Chat intro message for the gemma model" formatter_data="webui=AiChat">
+    Hi, I'm Leo. I'm a fully hosted AI assistant by Brave. I'm powered by Gemma 3 12B, a model created by Google DeepMind to be performant and applicable to many use cases. Gemma is Brave-hosted through our own secure infrastructure.<ph name="LINK_BEFORE">$1</ph>Learn more<ph name="LINK_AFTER">$2</ph>
```

Comment:
<img width="268" height="160" alt="Screenshot 2025-07-28 at 2 13 18 PM" src="https://github.com/user-attachments/assets/c1530401-7da9-4ab2-8cc4-53a53364b19d" />

You need to update to how other INTRO_MESSAGE look like in master, should be:
```
  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_GEMMA" desc="AI Chat intro message for the gemma model" formatter_data="webui=AiChat">
      Hi, I'm Leo. I'm a fully hosted AI assistant by Brave. I'm powered by Gemma 3 12B, a model created by Google DeepMind to be performant and applicable to many use cases. Gemma is Brave-hosted through our own secure infrastructure.<ph name="LINK_BEFORE">$1</ph>Learn more<ph name="LINK_AFTER">/$1</ph>
  </message>
```

Reply:
Also, we would need to update the learn more link: https://support.brave.app/hc/en-us/articles/26727364100493-What-are-the-differences-between-Leo-s-AI-Models to include gemma at some point.